modules/custom/az_dam/config/quickstart/core.entity_view_display.node.ib_demo_page.default.yml

```diff
@@ -0,0 +1,38 @@
+uuid: 6da96317-1d5f-4c66-ad69-71a16f079ef5
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.